src/pages/billing/Project.js

```diff
@@ -26,16 +28,21 @@ import { billingRoles } from 'src/pages/billing/List'
 const workspaceLastModifiedWidth = 150
 const workspaceExpandIconSize = 20
 
-const workspaceBillingStatusIconOrEmpty = (() => {
+const BillingAccountIcon = {
+  updating: { shape: 'sync', color: 'orange' },
+  done: { shape: 'check', color: colors.accent() },
+  error: { shape: 'warning-standard', color: 'red' }
+}
+
+const getBillingAccountIconOrEmpty = (() => {
   const size = 16
-  const blank = div({ style: { width: size } },
-    [div({ className: 'sr-only' }, ['Status'])])
-  return shape => shape ? icon(shape, { size }) : blank
+  const blank = div({ style: { width: size } }, [div({ className: 'sr-only' }, ['Status'])])
```

Comment:
Because this is returning the actual information for status, it feels a little weird to also be used for the header like this. I looked further down and saw it was also used for the status column on the workspaces list when everything is up to date. I think my preference would be for the text to say something like: `done` or for us to completely hide the column. Other than serving as a header for this column (which we want if there are icons), I think that this will cause noise if there's not some information there. `done` would provide a bit of information that makes it clear what this is reporting at least.

That would require pulling the header out into its own component, which I think would be my slight preference, just to make the distinction a bit clear between header and component (which follows the general repo pattern a bit more)

Reply:
Sure - I've conditionally hid the column (and padded the next) when no icons are required. Let me know what you think.